client/src/main/scala/drt/client/components/FlightComponents.scala

```diff
@@ -36,6 +36,11 @@ object FlightComponents {
         |Max: $max $apiPax""".stripMargin
   }
 
+  def paxTransferComponent(flight: Arrival) = <.div(
+    ^.className := "right",
+    s"${flight.TranPax.getOrElse(0)}"
```

Comment:
we should probably default to "-" or an empty cell, as our recent experience has shown us that a null value doesn't necessarily mean 0.